packages/amplication-notification-service/src/app.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Inject, Injectable } from "@nestjs/common";
+import { AmplicationLogger } from "@amplication/util/nestjs/logging";
+import { NovuService } from "./util/novuService";
+import { novuPackage } from "./util/novuPackage";
+import { subscribeUser } from "./notification-packages/subscribeUser";
+import { buildCompleted } from "./notification-packages/buildCompleted";
+
+const compose =
+  (...fns) =>
+  (x) =>
+    fns.reduce(async (y, f) => f(await y), x);
```

Comment:
Let's try to add some unit test here to get confidence in the solution

Reply:
It's a simple composition, nothing to test here